cloudweatherreport/datastore.py

```diff
@@ -92,8 +87,7 @@ def lock(self, path=None, timeout=5*60):
         # consistency to ensure it will be immediately visible to others,
         # and their lock files to us.
         lock_filename = '.lock.{}'.format(uuid4())
-        lock_filepath = self._path(path, lock_filename)
-        self.write(lock_filepath, '')
+        self.write(lock_filename, '')
```

Comment:
Upon further reflection, we have to lock the index anyway, which amounts to a global lock.  So we can just drop support for that path argument to the lock after all.